test/common/deployment_types.go

```diff
@@ -93,61 +99,61 @@ func getDeploymentConfiguration(deploymentName string, inst *integreatlyv1alpha1
 				{Name: "prometheus-operator", ExpectedReplicas: 1},
 			},
 		},
-		"rhmiOperatorDeploymentForRhmi2": Namespace{
+		"rhmiOperatorDeploymentForRhmi2": {
 			Name: RHMIOperatorNamespace,
 			Products: []Product{
 				{Name: "standard-authservice-postgresql", ExpectedReplicas: 1},
 			},
 		},
-		"rhmiOperatorDeploymentForManagedApi": Namespace{
+		"rhmiOperatorDeploymentForManagedApi": {
 			Name:     RHMIOperatorNamespace,
 			Products: []Product{},
 		},
-		"rhssoOperatorDeployment": Namespace{
+		"rhssoOperatorDeployment": {
 			Name: RHSSOOperatorNamespace,
 			Products: []Product{
 				{Name: "keycloak-operator", ExpectedReplicas: 1},
 			},
 		},
-		"solutionExplorerOperatorDeployment": Namespace{
+		"solutionExplorerOperatorDeployment": {
 			Name: SolutionExplorerOperatorNamespace,
 			Products: []Product{
 				{Name: "tutorial-web-app-operator", ExpectedReplicas: 1},
 			},
 		},
-		"upsOperatorDeployment": Namespace{
+		"upsOperatorDeployment": {
 			Name: UPSOperatorNamespace,
 			Products: []Product{
 				{Name: "unifiedpush-operator", ExpectedReplicas: 1},
 			},
 		},
-		"upsDeployment": Namespace{
+		"upsDeployment": {
 			Name: NamespacePrefix + "ups",
 			Products: []Product{
 				{Name: "ups", ExpectedReplicas: 1},
 			},
 		},
-		"rhssoUserOperatorDeployment": Namespace{
+		"rhssoUserOperatorDeployment": {
 			Name: RHSSOUserOperatorNamespace,
 			Products: []Product{
 				{Name: "keycloak-operator", ExpectedReplicas: 1},
 			},
 		},
-		"marin3rOperatorDeployment": Namespace{
+		"marin3rOperatorDeployment": {
 			Name: Marin3rOperatorNamespace,
 			Products: []Product{
 				{Name: "marin3r-controller-manager", ExpectedReplicas: 1},
 				{Name: "marin3r-controller-webhook", ExpectedReplicas: 2},
 			},
 		},
-		"marin3rDeployment": Namespace{
+		"marin3rDeployment": {
 			Name: Marin3rProductNamespace,
 			Products: []Product{
 				{Name: "prom-statsd-exporter", ExpectedReplicas: 1},
 				{Name: "ratelimit", ExpectedReplicas: 3},
 			},
```

Comment:
We could remove this since it's later assigned and only applies to managed api

Reply:
Removed. You are absolutely right.